crates/oxc_parser/examples/test.ts

```diff
@@ -0,0 +1,2 @@
+
+const someVariable: { [key in SomeEnum] } = { FOO: 3 };

```

Comment:
The case in #932 can be parsed now, but I am sure where should I add a new test case.

Reply:
I'm not certain you need it, the conformance tests should be enough